packages/experimental/Expander/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@fluentui-react-native/experimental-expander",
+  "version": "0.1.0",
+  "description": "A cross-platform Native Expander component using the Fluent Design System. Currently only implemented on windows",
+  "license": "MIT",
+  "author": "Microsoft <fluentuinativeowners@microsoft.com>",
+  "homepage": "https://github.com/microsoft/fluentui-react-native",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "just": "fluentui-scripts",
+    "clean": "fluentui-scripts clean",
+    "lint": "fluentui-scripts eslint",
+    "depcheck": "fluentui-scripts depcheck",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "verify-api": "fluentui-scripts verify-api-extractor",
+    "update-api": "fluentui-scripts update-api-extractor",
+    "windows": "react-native run-windows"
+  },
+  "dependencies": {
+    "@fluentui-react-native/component-cache": "^1.2.0",
+    "@fluentui-react-native/framework": "0.5.15"
+  },
+  "devDependencies": {
+    "@types/react-native": "^0.63.0",
+    "@uifabricshared/build-native": "^0.1.1",
+    "@uifabricshared/eslint-config-rules": "^0.1.1",
+    "react": "16.13.1",
+    "react-native": "^0.63.4",
+    "react-native-windows": "^0.63.0-0"
+  },
+  "peerDependencies": {
+    "react": "16.13.1",
```

Comment:
The version of React can change with newer versions of RN, so this should allow newer versions as well.